apps/meteor/client/views/omnichannel/directory/calls/CallTable.tsx

```diff
@@ -147,8 +148,10 @@ const CallTable: FC = () => {
 			const duration = moment.duration(callDuration / 1000, 'seconds');
 			return (
 				<Table.Row key={_id} tabIndex={0} role='link' onClick={(): void => onRowClick(_id, v?.token)} action qa-user-id={_id}>
-					<Table.Cell withTruncatedText>{fname}</Table.Cell>
-					<Table.Cell withTruncatedText>{Array.isArray(v?.phone) ? v?.phone[0]?.phoneNumber : v?.phone}</Table.Cell>
+					<Table.Cell withTruncatedText>{parseOutboundPhoneNumber(fname)}</Table.Cell>
```

Comment:
There could be rooms without `fname` out there. I can add a migration to fix them, but you can put a default value in here too (to avoid crashes)

Reply:
well, fname always will be part of VoIP rooms...

Reply:
Yep, but some old rooms (from the earlier stages of MVP) don't have that prop. If there's one of those on the wild, then the page may crash 👀 (for new rooms, this shouldn't be a problem)